services/core-api/app/api/now_applications/models/administrative_amendments/application_reason_code.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.associationproxy import association_proxy
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.schema import FetchedValue
+
+from app.api.utils.models_mixins import Base
+from app.extensions import db
+
+
+class ApplicationReasonCode(Base):
+    __tablename__ = 'application_reason_code'
+    application_reason_code = db.Column(db.String(3), nullable=False, primary_key=True)
```

Comment:
No need to specify nullable for primary keys

Reply:
done